components/sup/src/manager/service/spec.rs

```diff
@@ -123,47 +122,36 @@ pub trait IntoServiceSpec {
 
 impl IntoServiceSpec for protocol::ctl::SvcLoad {
     fn into_spec(&self, spec: &mut ServiceSpec) {
-        spec.ident = self.get_ident().clone().into();
-        if self.has_group() {
-            spec.group = self.get_group().to_string();
+        spec.ident = self.ident.clone().unwrap().into();
+        spec.group = self.group.clone().unwrap_or(DEFAULT_GROUP.to_string());
```

Comment:
@christophermaier this is the line that you care about 😢

Reply:
![tenor-115949343](https://user-images.githubusercontent.com/207178/40786096-3e8b166e-64b8-11e8-9bb2-75e6340289ee.gif)